app/src/entry-point.lisp

```diff
@@ -223,11 +223,14 @@ Copyright (c) 2016-2019 Rigetti Computing.~2%")
           (length qvm::*available-kernels*)
           (qvm::qubit-limit-for-using-serial-kernels))
   (let ((qvm-features
-          (list                         ; List of features
-                #+qvm-intrinsics
-                "qvm-intrinsics"
-                #+(and qvm-intrinsics avx2)
-                "avx2")))
+          (list*
+           ;; List of features
+           #+qvm-intrinsics
+           "qvm-intrinsics"
+           #+qvm-intrinsics
+           (if (sb-alien:extern-alien "avx2_supported" sb-alien:int)
```

Comment:
Is there a way to check if the alien exists before calling it, just to be safe?

Reply:
The only way to do this requires so much internals frobbing that the code for the check would bitrot much faster than, say, this alien disappearing in a future sbcl version.